buf/registry/module/v1beta1/label_service.proto

```diff
@@ -96,7 +97,9 @@ message ListLabelsRequest {
   // Once the resource is resolved, the following Labels are listed:
   //   - If a Module is referenced, all Labels for the Module are returned.
   //   - If a Label is referenced, this Label is returned.
```

Comment:
I think we probably need to handle filtering out the label the user supplied (if it doesn't match the review states filter below).

Reply:
Any particular reason? I don't think we should special case this.